src/components/ReportNavBar/styles.scss

```diff
@@ -0,0 +1,78 @@
+.report-navbar-container {
+  display: flex;
+  justify-content: space-between;
+  height: 64px;
+  width: 100%;
+  margin-top: 16px;
+  margin-bottom: 16px;
+  background-color: rgba(204,204,221,0.2);
+
+  .right-navbar {
+    display: flex;
+    margin: 11px 32px 11px 32px;
+    color: #667;
+    font-family: Arial;
```

Comment:
Change all font-family to 'DIN pro'

Reply:
Thanks @matthewacha, I'll implement that.